src/gui/mzroll/alignmentvizwidget.cpp

```diff
@@ -130,17 +130,36 @@ void AlignmentVizWidget::drawMessageBox(PeakGroup newGroup, PeakGroup group) {
 
 }
 
-float AlignmentVizWidget::calculateRsquare(PeakGroup group) {
-
-    float r2 = 0;
-
-    Q_FOREACH(Peak peak, group.getPeaks()) {
+float AlignmentVizWidget::calculateRsquare(PeakGroup newGroup,PeakGroup oldGroup) {
+
+    float SSres = 0;
+    float SStot=0;
+    float mean=0;
+
+    vector<mzSample*> oldSamples = getSamplesFromGroup(oldGroup);
+    vector<mzSample*> newSamples = getSamplesFromGroup(newGroup);
+    vector<float> newRts,oldRts;
+    for(int i=0;i<oldSamples.size();++i){
```

Comment:
@IpankajI Please take care of code formatting. It is not consistent.

Reply:
@shubhra-agrawal changed!